website/docs/r/endpoints_service.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "google"
+page_title: "Google: google_endpoints_service"
+sidebar_current: "docs-google-endpoints-service"
+description: |-
+  Creates and rolls out a Google Endpoints service.
+---
+
+# google_endpoints_service
+
+This resource creates and rolls out a Cloud Endpoints service using OpenAPI or gRPC.  View the relevant docs for [OpenAPI](https://cloud.google.com/endpoints/docs/openapi/) and [gRPC](https://cloud.google.com/endpoints/docs/grpc/).
+
+## Example Usage
+
+```hcl
+resource "google_endpoints_service" "openapi_service" {
+  service_name = "api-name.endpoints.project-id.cloud.goog"
+  project = "project-id"
+  openapi_config = "${file("openapi_spec.yml")}"
+}
+
+resource "google_endpoints_service" "grpc_service" {
+  service_name = "api-name.endpoints.project-id.cloud.goog"
+  project = "project-id"
+  grpc_config = "${file("service_spec.yml")}"
+  protoc_output = "${file("compiled_descriptor_file.pb")}"
+}
+```
+
+The example in `examples/endpoints_on_compute_engine` shows the API from the quickstart running on a Compute Engine VM and reachable through Cloud Endpoints, which may also be useful.
```

Comment:
I'd remove "which may also be useful" since that's implied, but you're welcome to keep it if you want